### ISSUE TITLE
Menu items lacking accessible expanded/collapsed state markup

### ISSUE THREAD

**Issue by jonels-msft** [readthedocs/sphinx_rtd_theme#955]
### Problem

The table of contents does not provide any markup beyond the "current" class to designate items that are open.
https://github.com/readthedocs/sphinx_rtd_theme/blob/fd6f31fc14e0594ef329ce37a4dd1848f81ab458/src/theme.js#L146
For accessibility, menu items should use standardized markup to indicate their state (expanded or collapsed).

### Reproducible Project

https://docs.citusdata.com/en/v9.3/

#### Error Logs/Results

Here is a video demonstrating the problem: https://www.youtube.com/watch?v=nIgVpPzhLEs

#### Expected Results

Markup should define a state, in conformance to [WCAG 4.1.2](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

### Environment Info

- Python Version: 3.7.7
- Sphinx Version: 1.4.9
- RTD Theme Version: 0.2.4

**Comment by Blendify** [comment]
Would adding an aria-label "Expanded" satisfy the issue?
